dts/bindings/i2s/st,stm32-sai.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2024 ZAL Zentrum für Angewandte Luftfahrtforschung GmbH
+# SPDX-License-Identifier: Apache-2.0
+
+description: STM32 I2S controller
+
+compatible: "st,stm32-sai"
+
+include: [i2s-controller.yaml, pinctrl-device.yaml]
+
+properties:
+  reg:
+    required: true
+
+  dmas:
+    required: true
+
+  dma-names:
+    required: true
```

Comment:
Please add a `description` here to list the available options for user.
Note this ends up being accessible [here](https://docs.zephyrproject.org/latest/build/dts/api/bindings/qspi/st%2Cstm32-qspi.html#dtbinding-st-stm32-qspi) for users